.gitignore

```diff
@@ -20,10 +20,8 @@ nbactions.xml
 .gradle/
 build/
 
-# maven stuff (to be removed when trunk becomes 4.x)
-*-execution-hints.log
-target/
-dependency-reduced-pom.xml
+# vscode stuff
```

Comment:
Does the comment add anything right now?

Reply:
It is fairly obvious what the line is for but all the other groups of lines have a comment so I figured I'd play along.

Reply:
Okay, I am fine with when in Rome....